docs/tests/integration/test_wazuh_db/test_wazuh_db.md

```diff
@@ -58,6 +58,8 @@ The **insert** and **clear** commands for `vuln_cves` table are tested:
 
 ### Checks chunks
 
+### Checks checksum range behavior
```

Comment:
??? 
![image](https://user-images.githubusercontent.com/23462183/123414361-15b5d600-d5b4-11eb-8290-b3c83541090b.png)

Reply:
I added a brief for all these legacy tests.